test/crd.go

```diff
@@ -18,28 +18,24 @@ package test
 // crd contains functions that construct boilerplate CRD definitions.
 
 import (
-	"fmt"
-
-	channelsV1alpha1 "github.com/knative/eventing/pkg/apis/channels/v1alpha1"
-	feedsV1alpha1 "github.com/knative/eventing/pkg/apis/feeds/v1alpha1"
-	flowsV1alpha1 "github.com/knative/eventing/pkg/apis/flows/v1alpha1"
-	"github.com/knative/serving/pkg/apis/serving/v1alpha1"
+	sourcesv1alpha1 "github.com/knative/eventing-sources/pkg/apis/sources/v1alpha1"
+	"github.com/knative/eventing/pkg/apis/eventing/v1alpha1"
+	servingv1alpha1 "github.com/knative/serving/pkg/apis/serving/v1alpha1"
 	corev1 "k8s.io/api/core/v1"
```

Comment:
I would ❤️ if the import changes were a separate PR from the addition of the new CRD types to the test.